setup.py

```diff
@@ -23,7 +23,7 @@ def read(fname):
 
 
 setup(name="sagemaker",
-      version="1.9.0",
+      version="1.9.1",
```

Comment:
Why do we change the version here? We are not going to release.

Reply:
Yeah. You are right. I will remove this. We do not need to release to fix the tests.